actions/commit-and-push/action.yaml

```diff
@@ -0,0 +1,37 @@
+name: "Commit and push"
+description: "Commits all the changed files and pushes them to the repository"
+
+inputs:
+  ref:
+    description: "The ref name to commit and push the files on."
+    required: true
+  commit-message:
+    description: "The commit message."
+    required: true
```

Comment:
Do you think a default commit message makes sense here? Something like `Committing CI changes`?

Reply:
For the action here I would keep it like this. In the main workflow, I am passing a hardcoded message. It's nice to have the freedom to choose the message :smile: :dove: